### PR TITLE
add exclude operation for filter elastic result

### PR DESCRIPTION
**Pull request by hnhbwlp**
add exclude operation for data query filter

### CODE REVIEW

**Review by linuzb — APPROVED** [review]
LGTM

File: internal/grafanadi/utils/utils.go [hunk 1]
```diff
@@ -91,8 +91,16 @@ func AggregationFromParams(index, doctype, timeAttr, filters, lte, gte, terms, p
 		if len(fAry[1]) == 0 {
 			continue
 		}
-		boolSearch = boolSearch.Filter(elastic.NewTermsQuery(fAry[0], fAry[1]))
-		//boolSearch = boolSearch.Must(elastic.NewTermsQuery(fAry[0], fAry[1]))
+		// add operation for query
+		if strings.HasPrefix(fAry[1], "!") {
+			fVal := strings.TrimPrefix(fAry[1], "!")
+			if len(fVal) == 0 {
+				continue
```
Comment (larryck) [inline internal/grafanadi/utils/utils.go:98]: Empty values will be skipped. LGTM.

**Review by larryck — APPROVED** [review]

**Comment by larryck** [comment]
This would be a useful feature. For the DebugSLO module, currently we can filter and see the delivery statistics at cluster/namespace/result level. This feature gives us the ability to filter data without a specific item. A widely used scenario could be to filter out all the success pods and only look at the distribution of error pods.

**Review by d3c3mber — COMMENTED** [review]
LGTM

**Pull request merged**